### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [jaszan/cc_teacozy#4]
https://github.com/jaszan/cc_teacozy/blob/8736ae1f8cd4a4fe452e3e767c2d8881a518045b/index.html#L23

Every image should have descriptive alt text.  Without it, your HTML doesn't pass validation.  Also, alt text helps the visually impaired.  The screen readers they use will use alt text to describe the image.
